sdx-connector/src/main/java/com/sequenceiq/cloudbreak/saas/sdx/SaasSdxService.java

```diff
@@ -0,0 +1,84 @@
+package com.sequenceiq.cloudbreak.saas.sdx;
+
+import static com.cloudera.thunderhead.service.sdxsvcadmin.SDXSvcAdminProto.InstanceHighLevelStatus.Value.UNHEALTHY;
+import static com.sequenceiq.cloudbreak.auth.altus.model.Entitlement.CDP_SAAS_SDX_INTEGRATION;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.inject.Inject;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.cloudera.thunderhead.service.sdxsvcadmin.SDXSvcAdminProto;
+import com.sequenceiq.cloudbreak.auth.altus.model.Entitlement;
+import com.sequenceiq.cloudbreak.auth.crn.Crn;
+import com.sequenceiq.cloudbreak.auth.crn.CrnResourceDescriptor;
+import com.sequenceiq.cloudbreak.logger.MDCUtils;
+import com.sequenceiq.cloudbreak.saas.client.sdx.GrpcSdxSaasClient;
+import com.sequenceiq.cloudbreak.saas.sdx.polling.PollingResult;
+
+@Service
+public class SaasSdxService extends AbstractSdxService<SDXSvcAdminProto.InstanceHighLevelStatus.Value> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SaasSdxService.class);
+
+    @Inject
+    private GrpcSdxSaasClient grpcSdxSaasClient;
+
+    @Override
+    public CrnResourceDescriptor crnResourceDescriptor() {
+        return CrnResourceDescriptor.SDX_SAAS_INSTANCE;
+    }
+
+    @Override
+    public void deleteSdx(String sdxCrn, Boolean force) {
+        if (isPlatformEntitled(Crn.safeFromString(sdxCrn).getAccountId())) {
+            LOGGER.info("Calling deleteInstance for SDX SaaS instance {}", sdxCrn);
+            grpcSdxSaasClient.deleteInstance(MDCUtils.getRequestId(), sdxCrn);
+        }
```

Comment:
please log if not entitled

Reply:
it is logged in EntitlementService, it would be a duplicated log